### PR TITLE
AlwaysTrust: Add options for each popup

### PR DESCRIPTION
**Pull request by trwy7**
This should allow you to change what popups get removed with AlwaysTrust, defaulting to all, before you would need to have both popups hidden. This addon allows you to choose which popup(s) to remove, instead of all of them. I have checked this with my own Vencord installation before making this PR.

### CODE REVIEW

**Comment by trwy7** [comment]
Bumping as I added a more clear title

**Comment by trwy7** [comment]
Checks seem to be fixed now.

File: src/plugins/alwaysTrust/index.ts [hunk 1]
```diff
@@ -16,27 +16,46 @@
...
+import { definePluginSettings } from "@api/Settings";
 import { Devs } from "@utils/constants";
-import definePlugin from "@utils/types";
+import definePlugin, { OptionType } from "@utils/types";
+
+const settings = definePluginSettings({
+    domain: {
+        type: OptionType.BOOLEAN,
+        default: true,
+        description: "Remove the untrusted domain popup when clicking untrusted domains",
```
Comment (Vendicated) [inline src/plugins/alwaysTrust/index.ts:27]:
```suggestion
        description: "Remove the untrusted domain popup when opening links",
```

File: src/plugins/alwaysTrust/index.ts [hunk 2]
```diff
@@ -16,27 +16,46 @@
...
+        restartNeeded: true
+    },
+    file: {
+        type: OptionType.BOOLEAN,
+        default: true,
+        description: "Remove the suspicious file popup when downloading executable files",
```
Comment (Vendicated) [inline src/plugins/alwaysTrust/index.ts:33]:
```suggestion
        description: "Remove the 'Potentially Dangerous Download' popup when opening links",
```

**Comment by trwy7** [comment]
Commits have been added.

**Comment by Vendicated** [comment]
thanks!

**Pull request merged**